SkyAPM/SkyAPM-dotnet
# add kafka reporter

Please answer these questions before submitting pull request

- Why submit this pull request?
New feature provided

- How to test?
grpc server: `set ASPNETCORE_ENVIRONMENT=Development && set ASPNETCORE_HOSTINGSTARTUPASSEMBLIES=SkyAPM.Agent.AspNetCore && sample\grpc\SkyApm.Sample.GrpcServer\bin\Release\net6.0\SkyApm.Sample.GrpcServer.exe`
grpc client: `set ASPNETCORE_ENVIRONMENT=Development && set ASPNETCORE_HOSTINGSTARTUPASSEMBLIES=SkyAPM.Agent.AspNetCore && sample\grpc\SkyApm.Sample.GrpcClient\bin\Release\net6.0\SkyApm.Sample.GrpcClient.exe`

SEE ALSO #559
